index.html

```diff
@@ -145,44 +145,60 @@ <h4 data-ja="一次情報源">Primary Data Sources</h4>
     if(typeof drawMap === 'undefined'){
       return
     }
-    
+
     initDataTranslate()
     drawMap()
-    
+
+    var draws = 0;
```

Comment:
Maybe `pageDraws` to be slightly more specific? Especially with a global variable.

Reply:
done :+1: